api/client/identityfile.go

```diff
@@ -95,7 +108,23 @@ func ReadIdentityFile(path string) (*IdentityFile, error) {
 		return nil, trace.Wrap(err)
 	}
 	defer r.Close()
-	return decodeIdentityFile(r)
+
+	ident, err := decodeIdentityFile(r)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	// did not find the SSH certificate in the file? look in a
+	// separate file with -cert.pub prefix
+	if len(ident.Certs.SSH) == 0 {
+		certFn := path + "-cert.pub"
```

Comment:
nit: move this suffix into a constant somewhere
we already have a const in `lib/auth/keystore.go` btw